src/Modules/AddTodo/components/index.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { ChangeEvent, FormEvent } from "react";
+import { NavLink } from "react-router-dom";
+
+import { ROUTES } from "../../../appContants";
+import { TASK_STATE } from "../../../constant";
+
+interface IProps {
+  title: string;
+  titleHandler: (e: ChangeEvent<HTMLInputElement>) => void;
```

Comment:
create a separate file named constants.ts and create a type alias for these type and import it here